woocommerce/woocommerce-ios
# [Woo POS] Update toolbar design based on high-fidelity design

Closes: #12971 


## Description


This PR includes design updates mainly on the toolbar to match the latest M1 high-fidelity design p91TBi-bot-p2. The dashboard background was also updated based on the design, but I didn't go very far into further design updates on the product selector & cart views in case it's already underway. I exported the swap icon because it looks quite different from the SF Symbol `arrow.left.arrow.right`.


## Testing information


Prerequisite: the store is eligible for POS and the feature switch is enabled in Menu > Settings > Experimental Features > POS

* Launch app
* Enter POS mode --> the toolbar should match the design in the disconnected state
* Tap `Connect now` and proceed to connect to a reader --> the toolbar should match the design in the connected state

## Screenshots


Reader connected:
<img width="1700" alt="Screenshot 2024-06-11 at 10 24 46 AM" src="https://github.com/woocommerce/woocommerce-ios/assets/1945542/70ff281b-0a64-49f4-b343-63287352ab5f">

Reader disconnected:
<img width="1701" alt="Screenshot 2024-06-11 at 10 26 28 AM" src="https://github.com/woocommerce/woocommerce-ios/assets/1945542/5114b47f-1e1f-4b6d-a3e4-fc31ae632cd0">


---
